docs/book/src/cronjob-tutorial/testdata/project/go.mod

```diff
@@ -1,6 +1,6 @@
 module tutorial.kubebuilder.io/project
 
-go 1.17
+go 1.18
```

Comment:
The docs are not update accordingly
These are projects generated with the tool.
Could we revert all changes under docs/book and then update it in a follow-up?
See that we need to regenerate the samples used in the docs with scripts and ensure that it is properly updated.

Reply:
I'm not sure I understood...

Reply:
@camilamacedo86 now I understand...
OMG, this is a nightmare 😖

Are these dependencies so important? also, can't we split it into a different PR?
I suggest I'll just update the deps and keep the rest the same (otherwise, it's just crazy without the knowledge of what's happening here)

Reply:
@AlmogBaku, 

My suggestion here is:
- Revert all changes done in the directory: [docs/](https://github.com/kubernetes-sigs/kubebuilder/pull/2561/files/6f233b6a3edab85f9486e2c18dc24176a10d6eb7#diff-93830d0e4cc14bb4d52a3cce90f1ade789b7293dab0f75abaabb7afb7de4ebf3)
- We do a PR to just updated the docs with all master changes done so far

Reply:
done